ledger/daml-on-sql/README.rst

```diff
@@ -13,6 +13,8 @@
 
 *Daml Driver for PostgreSQL* is a PostgreSQL-based Daml ledger implementation.
 
+The Community Edition, distributed as a JAR file called `daml-on-sql-<version>.jar`, can be obtained from the `Daml GitHub Releases Page <https://github.com/digital-asset/daml/releases>`__.
```

Comment:
that’s true for both CE and EE. The EE only provides a docker image in addition to the JAR on GH releases.